.circleci/config.yml

```diff
@@ -69,8 +69,15 @@ jobs:
       - run:
           name: Test of hello-visualization
           command: |
-            until $(curl --output /dev/null --silent --head --fail http://localhost:8080); do
+            set +e
+            RETRIES=0
+            while [[ $RETRIES -lt 30 ]]; do
+              curl --output /dev/null --silent --head --fail http://localhost:8080
+              if [[ $? -eq 0 ]]; then
+                break
+              fi
               printf '.'
```

Comment:
Does this `printf` add any value? Maybe better to echo something like `Web server is up`

Reply:
Agree with Web server up or ready would be better!